app-torcx/docker/files/docker-1.12.6-manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+    "kind": "image-manifest-v0",
+    "value": {
+        "bin": [
+            "/bin/containerd",
+            "/bin/containerd-shim",
+            "/bin/ctr",
+            "/bin/docker",
+            "/bin/docker-containerd",
+            "/bin/docker-containerd-shim",
+            "/bin/docker-proxy",
+            "/bin/docker-runc",
+            "/bin/dockerd",
+            "/bin/runc"
+        ],
+        "service": [
+            "/lib/systemd/system/containerd.service",
+            "/lib/systemd/system/docker.service",
+            "/lib/systemd/system/docker.socket",
```

Comment:
I was planning to only have service (and dependency units) in here, but you are right that sockets (and timers and some other things) go under `system/` as well. Do we want to a) split a separate section for sockets b) rename "service" into "system" or something equally generic c) do nothing?

The rationale for having separate sections in this assets manifest is to (possibly later) be able to aggregate metadata about torcx addons without having to unpack every single image.